src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx

```diff
@@ -21,12 +21,14 @@ import { usePartygoers } from "hooks/users";
 
 import UserList from "components/molecules/UserList";
 
-import { RoomModalOngoingEvent, ScheduleItem } from "../";
+import { RoomModalOngoingEvent, ScheduleItem } from "..";
```

Comment:
Not sure if we want to leave it as `..`. If so, should we also stick to `.`, instead of `./`?

Reply:
I don't actually have a strong opinion one way or the other about this. I wonder if there are best practices/linting rules that prefer one over the other?